filesystem/etc/rc.local

```diff
@@ -58,12 +59,12 @@ esac
 
 # If running libnetwork plugin in a separate container, CALICO_LIBNETWORK_ENABLED would be false.
 # CALICO_LIBNETWORK_ENABLED is "false" by default. It can be set by passing `--libnetwork` flag while starting the calico/node via calicoctl
-if [ "$CALICO_LIBNETWORK_ENABLED" == "true" ]; then
+if [ "$CALICO_LIBNETWORK_ENABLED" = "true" ]; then
 	echo "Starting libnetwork service"
 	cp -a /etc/service/available/libnetwork  /etc/service/enabled/
 fi
 
-if [ "$CALICO_DISABLE_FILE_LOGGING" == "true" ]; then
+if [ "$CALICO_DISABLE_FILE_LOGGING" = "true" ]; then
```

Comment:
These changes is a result of me debugging borked tests as a result of switching to Debian. 

I don't think these are strictly necessary (these ultimately were not the causes of the borked tests). However, running `shellcheck` on this (similar to what I did on [CNI Plugin base image refactor](https://github.com/projectcalico/cni-plugin/pull/708/files#diff-282edec51ec64b45bd3e29ddced5e78f)) showed the above recommended changes.